src/com/hp/ppm/integration/rally/RallyIntegrationConnector.java

```diff
@@ -0,0 +1,49 @@
+package com.hp.ppm.integration.rally;
```

Comment:
please change the java package from 
com.hp.ppm.integration.rally 
to
com.ppm.integration.agilesdk.connector.agilecentral